docker-compose-using-tube-cheese/README.md

```diff
@@ -0,0 +1,31 @@
+# Use sozu as reverse-proxy with docker-compose (using tube-cheese)
+
+First, you have to install [docker-compose](https://docs.docker.com/compose/install/) to execute this demo
+
+On OSX, you can use Docker for Mac with success.
+
+We have to add the domain names to our host file. We need to know the machine IP and add it to the /etc/hosts file
```

Comment:
You are right. It is confusing. I will try to be more short and concise.
Thanks